lib/iris/analysis/__init__.py

```diff
@@ -1740,9 +1743,9 @@ def interp_order(length):
 #
 COUNT = Aggregator(
     "count",
-    iris._lazy_data.non_lazy(_lazy_count),
+    _count,
     units_func=lambda units: 1,
-    lazy_func=_lazy_count,
+    lazy_func=_build_dask_mdtol_function(_count),
```

Comment:
I think it also needs explaining that `_count` can function as a (lazy-preserving) dask operation as well, 
purely due to _how_ it is implemented.
Or possibly, that it is better said where it is defined ?